siac/walletcmd.go

```diff
@@ -162,6 +169,9 @@ will be sent to your wallet.`,
 
 const askPasswordText = "We need to encrypt the new data using the current wallet password, please provide: "
 
+const newPasswordText = "Please provide the current password of the wallet: "
+const oldPasswordText = "Please provide the new password for the wallet: "
+
```

Comment:
Surely "Current password" and "New password" are clear enough? Users are familiar with password rotation.

`askPasswordText` is a bit long-winded too, although that's a less familiar situation.